index.html

```diff
@@ -19,7 +19,7 @@
             <img src="IMG1/icon.png" alt="Bars">
         </nav>
     </header>
-    <section>
+    <section class="first-section">
             <div class="header-container">
                 <h1>Winfred Edube</h1>
                 <h2>Software Developer</h2>
```

Comment:
- [ ] In this section, your `Software Developer` text is overlapping the `Winfred Edube` text. Please make sure to fix this issue.
Here's an image for reference:
![Captura de pantalla 2022-02-20 013130](https://user-images.githubusercontent.com/59100477/154823833-ec45f06e-81da-40fb-bff4-9ae43c62cf06.jpg)